src/main/java/com/alkemy/ong/exception/OngRequestException.java

```diff
@@ -0,0 +1,7 @@
+package com.alkemy.ong.exception;
+
+public class OngRequestException extends RuntimeException {
+    public OngRequestException(String msg, Throwable cause) {
```

Comment:
No está mal tener el Throwable, aunque nosotros no lo vamos a usar. Asi que podes tener además del msg (que le vamos a cambiar a `message` en el nombre del método) el campo code. Entonces cuando vos tiras una exception, tiras `throw new OngRequestException("User not found", "not.found");` y en el handler, te armas tu ApiErrorDTO con los dos campos de la exception, que se mapean con tus campos del DTO.